benchmark/assert/deepequal-map.js

```diff
@@ -24,15 +24,15 @@ function benchmark(method, n, values, values2) {
   const deepCopy = JSON.parse(JSON.stringify(values2 ? values2 : values));
   const expected = new Map(deepCopy);
   bench.start();
-  for (var i = 0; i < n; ++i) {
+  for (let i = 0; i < n; ++i) {
     method(actual, expected);
   }
   bench.end(n);
 }
 
 function main({ n, len, method, strict }) {
   const array = Array(len).fill(1);
-  var values, values2;
+  let values, values2;
```

Comment:
nit: these variables can be moved as `const` inside each case (since they're strictly used within the scope of each case)